docs/src/hugo/config.toml

```diff
@@ -1,8 +1,11 @@
-baseurl = "http://http4s.org/"
+baseurl = "http://http4s.org/v0.15"
```

Comment:
This is another hardcoding of the version I'd love to get rid of, but it merely shifts from menu.tut, so it's not worse.